WinUXTheme.m

```diff
@@ -57,7 +57,9 @@ + (void) initialize
   // Inserting WIN32VSImageRep class for themed images.
   [NSImageRep registerImageRepClass:[WIN32VSImageRep class]];
 
-
+  // See WinNSTabView.m for implemented usage...
+  [[NSUserDefaults standardUserDefaults] registerDefaults:@{@"GSMinimumTabHeight" : @12.0 }];
```

Comment:
I think this notation will only work with clang. Is this theme otherwise usable with gcc? In that case we should use the old notation here.